tree/ntuple/v7/test/ntuple_minifile.cxx

```diff
@@ -459,3 +464,43 @@ TEST(MiniFile, DifferentTKeys)
    auto ntuple = RNTupleReader::Open("Events", fileGuard.GetPath());
    EXPECT_EQ(1, ntuple->GetNEntries());
 }
+
+TEST(MiniFile, SchemaInfo)
+{
+   FileRaii fileGuardProper("test_ntuple_minifile_schema_info_proper.root");
+   FileRaii fileGuardSimple("test_ntuple_minifile_schema_info_simple.root");
+
+   std::vector<TVirtualStreamerInfo *> streamerInfos;
+   streamerInfos.emplace_back(TClass::GetClass("TVector2")->GetStreamerInfo());
+   streamerInfos.emplace_back(TClass::GetClass("TVector3")->GetStreamerInfo());
+
+   auto file = std::unique_ptr<TFile>(TFile::Open(fileGuardProper.GetPath().c_str(), "RECREATE"));
+   auto writerProper = RNTupleFileWriter::Append("MyNTuple", *file, RNTupleWriteOptions::kDefaultMaxKeySize);
+   writerProper->UpdateStreamerInfos(streamerInfos);
+   writerProper->Commit();
+   file->Close();
+
+   auto writerSimple =
+      RNTupleFileWriter::Recreate("ntpl", fileGuardSimple.GetPath(), 0, RNTupleFileWriter::EContainerFormat::kTFile,
+                                  RNTupleWriteOptions::kDefaultMaxKeySize);
+   writerSimple->UpdateStreamerInfos(streamerInfos);
+   writerSimple->Commit();
```

Comment:
Maybe put this into a C++ scope to make sure the `RNTupleFileWriter` are destructed?